manifests/m/Microsoft/DevHome/0.503.261.0/Microsoft.DevHome.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.503.261.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.22000.0

```

Comment:
```suggestion
MinimumOSVersion: 10.0.22621.0
```

Reply:
From AppxBundleManifest.xml

```xml
		<Package Type="application" Version="0.503.261.0" Architecture="x86" FileName="DevHome-x86.msix" Offset="278577046" Size="129502986">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
		<Package Type="application" Version="0.503.261.0" Architecture="x64" FileName="DevHome-x64.msix" Offset="138258783" Size="140318193">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
		<Package Type="application" Version="0.503.261.0" Architecture="arm64" FileName="DevHome-arm64.msix" Offset="3703639" Size="134555074">
			<Resources>
				<Resource Language="EN-US"/>
				<Resource Language="DE-DE"/>
				<Resource Language="ES-ES"/>
				<Resource Language="FR-FR"/>
				<Resource Language="IT-IT"/>
				<Resource Language="JA-JP"/>
				<Resource Language="KO-KR"/>
				<Resource Language="PT-BR"/>
				<Resource Language="QPS-PLOC"/>
				<Resource Language="QPS-PLOCA"/>
				<Resource Language="QPS-PLOCM"/>
				<Resource Language="RU-RU"/>
				<Resource Language="ZH-CN"/>
				<Resource Language="ZH-TW"/>
			</Resources>
			<b4:Dependencies>
				<b4:TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.22000.0" MaxVersionTested="10.0.22000.0"/>
			</b4:Dependencies>
		</Package>
```

For non-stub packages, I believe the MinOS version is `10.0.22000.0`